ckanext/data_qld/actions.py

```diff
@@ -98,8 +101,10 @@ def _undictize_datarequest_basic(datarequest, data_dict):
 
 def _undictize_datarequest_closing_circumstances(datarequest, data_dict):
     if tk.h.closing_circumstances_enabled:
-        datarequest.close_circumstance = data_dict.get('close_circumstance') or None
-        datarequest.approx_publishing_date = data_dict.get('approx_publishing_date') or None
+        datarequest.close_circumstance = data_dict.get(
+            'close_circumstance') or None
```

Comment:
This could be written more concisely as `data_dict.get('close_circumstance', None)`

Reply:
white space change flagged this in their pr, this is not related to their work.